src/utils/formatting.ts

```diff
@@ -73,9 +73,22 @@ export const liveChannelsURL = (playlistId: string, channelId?: string, play = f
   });
 };
 
-export const episodeURL = (episode: PlaylistItem, seriesId?: string, play: boolean = false, playlistId?: string | null) =>
-  addQueryParams(mediaURL(episode, playlistId, play), {
+export const episodeURL = ({
+  episode,
+  seriesId,
+  play,
+  playlistId,
+  mediaId,
```

Comment:
Why did you add a param for mediaId? Even if you need it, won't it be available on the episode PlaylistItem?